src/js/views/Certificates/layout/MassActions.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import { Grid, Box, Typography, Button, IconButton } from '@material-ui/core';
+import { VerifiedUser, Close, Delete, Star } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+import { useMassActionsStyles } from './style';
+
+const MassActions = ({
```

Comment:
The only button that should be on this component is delete all. Users cannot add certificates to favorites and the create certificates button is meaningless here.